hvpy/api_groups/movies/download_movie.py

```diff
@@ -0,0 +1,35 @@
+from typing import Optional
+
+from hvpy.io import HvpyParameters, OutputType
+
+
+class downloadMovieInputParameters(HvpyParameters):
+    """
+    Handles the input parameters of the ``downloadMovie`` API.
+
+    Attributes
+    ----------
+    {Shared}
+    id
+        Unique movie identifier, returned as a response by the ``queueMovie`` endpoint request.
+    format
+        Movie format.
+    hq
+        Download high quality movie file.
+        Defaults to `False`, optional.
+
+    References
+    ----------
+    * `<https://api.helioviewer.org/docs/v2/api/api_groups/movies.html#id9>`__
+    {Shared}
+    """
+
+    id: str
+    format: str
+    hq: Optional[bool] = False
```

Comment:
```suggestion
    hq: bool = False
```
`Optional` is only if the value can be set to `None`, it does not mean if its optional from a user point of view.

Reply:
Okay, understood

Reply:
No. You set the default value, that makes it optional.